external/fv3fit/fv3fit/emulation/keras.py

```diff
@@ -45,6 +45,6 @@ def score_model(model: tf.keras.Model, data: Mapping[str, tf.Tensor],) -> Scorin
         ``model``.
     """
     model = fv3fit.keras.adapters.ensure_dict_output(model)
-    prediction = model(data)
+    prediction = model.predict(data, batch_size=8192)
```

Comment:
The previous eager prediction step would OOM using 80,000 samples during scoring.  The `batch_size` of `.predict()` defaults to 32, so I chose a size that would run quickly and not use too much memory.